apps/e2e/src/trigger-client/fullSanity.ts

```diff
@@ -22,6 +25,11 @@ async function fullSanity(context: RunContext) {
   const signer = context.getWallet();
   const chainId = await getChainIdFromSigner(signer);
 
+  const engineClient = new EngineClient({
```

Comment:
`await clearinghouse.getOrderbook(ethProductId);` did the same thing right?

Reply:
clearinghouse no longer has `getOrderbook` which is why the sanity checks were broken in the first place

Reply:
Ah got it!

Reply:
the `OffchainExchange` contract has a `getVirtualBook` function that gets the books, but since we don't instantiate offchain exchange at all decided to just use engine instead for this